README.md

```diff
@@ -38,10 +38,10 @@ Overview of the main packages maintained in this repository:
 | [@kadena/cryptography-utils][32]     | [![version][34]][33] |
 | [@kadena/graph][35]                  | [![version][37]][36] |
 | [@kadena/hd-wallet][38]              | [![version][40]][39] |
-| [@kadena/kadena-cli][41]             | [![version][43]][42] |
```

Comment:
was this removal intentional?

Reply:
No this is all done by the tooling we use. I can try to put it back.

Reply:
Yes the package has been set to private like two weeks ago.